src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -169,10 +169,12 @@ public void keyboardShortcutsTest() {
         // test shortcut on repo dropdown
         doubleClick(repoSelectorComboBox);
         pushKeys(SWITCH_DEFAULT_REPO);
-        waitUntilNodeAppears("#dummy/dummy_col1_1");
+        // wait for issue 9 to appear then click on it
+        // issue 9 is chosen instead of issue 10 due to a problem with finding issue 10's node
```

Comment:
Not too sure, but as of now, in waitUntilNodeAppears, when it tries to find issue 10's node the second time (first time being unable to find, so it sleeps and checks again), it fails with an illegal argument exception. The code on our end seems to be fine, but when testfx's findByCssSelector is triggered, it throws an exception at line 443.

Should these details be in the comment?